src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -318,3 +325,43 @@ def github_comment_reactions():
             continue
 
         metrics.incr("github_pr_comment.comment_reactions.success")
+
+
+# TODO(adas): Change the client typing to allow for multiple SCM Integrations
+def safe_for_comment(
```

Comment:
we should probably also check if `pullrequest_resp["state"] == "open"` to make sure it's an open PR. it's possible that the PR is closed between when the task is queued and when it's executed